frontend/src/lib/WidgetStateManager.ts

```diff
@@ -193,6 +196,45 @@ export class WidgetStateManager {
     })
   }
 
+  public sendUploadFileMessage(
+    widgetId: string,
+    name: string,
+    lastModified: number,
+    data: Uint8Array
+  ): void {
+    // We send the file in severals chunks to avoid server webservices freezing.
+
+    // The default file size allowed by server config is 50MB.
+    // The best message size to don't freeze the server is 10MB
+    const chunkByteLimit = 10 * 1e6 //10MB
```

Comment:
What if the user sets the limit to 5MB? Does this break anything?

Reply:
No, we have a validation to check if the widget accept the file first